generators/app/templates/src/app/screens.ejs

```diff
@@ -6,37 +6,29 @@ import { <% if (features.tabs) { %>createBottomTabNavigator, <% } %>createStackN
 
 import Routes from '../constants/routes';
 import { stackNavConfig, screensNavOptions<% if (features.tabs) { %>, tabNavConfig<% } %> } from '../config/navigation';
+import { inferRoute } from '../utils/navUtils';
 
 <%_ if(features.login) { _%>
 import InitialLoading from './screens/InitialLoading';
 import Login from './screens/Login';
 <%_ } _%>
 import Home from './screens/Home';
 
+const Tab1 = Home;
+const Tab2 = Home;
```

Comment:
You must do this only if `features.tabs` is true because if not, the constants `Tab1` and `Tab2` aren't going to be used:
```
<%_ if(features.tabs) { _%>
const Tab1 = Home;
const Tab2 = Home;
<%_ } _%>
```